lab/src/Microsoft.ServiceModel.Syndication/src/SyndicationItemFormatter.cs

```diff
@@ -187,12 +180,12 @@ protected static bool TryParseElement(XmlReader reader, SyndicationPerson person
 
         protected static void WriteAttributeExtensions(XmlWriter writer, SyndicationItem item, string version)
         {
```

Comment:
Dangerous method and it will return when WriteAttributeExtensionsAsync is still writing to writer. This can cause state corruption/exceptions to be thrown in writer